src/utils/processSystem.cpp

```
@@ -10,23 +20,23 @@ ProcessSystem::~ProcessSystem()
 {
 
 }
-ProcessSystem::ProcessSystem(uint threadNumber)
+ProcessSystem::ProcessSystem(uint _threadNumber)
 {
-    ProcessSystem::threadNumber = threadNumber;
-    ProcessSystem::isRun = false;
+    threadNumber = _threadNumber;
```

cppcheck

[performance] Variable 'threadNumber' is assigned in constructor body. Consider performing initialization in initialization list.

(useInitializationList)